AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -161,6 +162,47 @@ extension CardViewController {
             return holderNameItem
         }()
 
+        internal lazy var coBadgedCardItem: FormCoBadgedCardItem = {
+            let item = FormCoBadgedCardItem(
+                title: localizedString(
+                    .creditCardDualBrandTitle,
+                    configuration.localizationParameters
+                ),
+                subtitle: localizedString(
+                    .creditCardDualBrandDescription,
+                    configuration.localizationParameters
+                ),
+                selectableFormItems: [],
+                style: .init(
+                    title: configuration.style.sectionHeader,
+                    subtitle: configuration.style.footnoteLabel
+                )
+            )
+            item.style.subtitle.textAlignment = .left
+            item.identifier = ViewIdentifierBuilder.build(
+                scopeInstance: self,
+                postfix: "coBadgedCardItem"
+            )
+            return item
+        }()
+
+        internal func selectableFormItems(from brands: [CardBrand]) -> [SelectableFormItem] {
+            brands.map { brand in
+                let selectableItem = SelectableFormItem(
+                    title: brand.type.name,
+                    imageUrl: numberContainerItem.numberItem.cardTypeLogos.filter { $0.type == brand.type }.first?.url,
```

Comment:
Moving this outside the initializer to their own constant declaration would improve readability:

```swift
let imageUrl = numberContainerItem.numberItem.cardTypeLogos.filter { $0.type == brand.type }.first?.url
```

```swift
let isSelected = brand.type.rawValue == brands.first?.type.rawValue ? true : false
```